Sphinxsetup.sh

```diff
@@ -38,32 +48,24 @@ if [ "$(python --version)" == "Python 3.6.9" ]; then
     GET_PIP_URL="https://bootstrap.pypa.io/pip/3.6/get-pip.py"
 fi
 
-curl "$GET_PIP_URL" -o get-pip.py
-python3 get-pip.py
-rm -f get-pip.py
+if [ ${DISTRIBUTION_CODENAME} != 'lunar' ]; then
+  curl "$GET_PIP_URL" -o get-pip.py
```

Comment:
Why not install pip from apt?

Reply:
because you don't have the latest version so it can have some side effect

Reply:
But then you can just 
`python3 -m pip install --upgrade pip` as it recommends in the terminal.

Reply:
You cannot update systems package with pip, so apt package stay at the repo version.